internal-packages/ui/style.css

```diff
@@ -20,6 +20,8 @@
 	--color-elevated-surface-background: #2f343e;
 	--color-surface-background: #2f343e;
 	--color-background: #00020b;
+	/* semantic base background (can be overridden per app) */
+	--color-bg: var(--color-background);
```

Comment:
Is there a possibility that in the future we will be able to set different colors for "background" and "bg"? That would be counterintuitive, so I think it would be better to give them different names. Also, having "background" and "bg" with the same color is confusing, so I think it should be one of them.